server/src/ClientToDBAPI/ContactedPeople/mainRoute.ts

```diff
@@ -143,21 +144,32 @@ ContactedPeopleRoute.post('/interactedContacts', (request: Request, response: Re
     logger.info({
         service: Service.SERVER,
         severity: Severity.LOW,
-        workflow,
+        workflow: `Saving ${isSingleContact ? 'single': 'all'} contact${isSingleContact ? '' : 's'}`,
         step: `launching graphql API request with parameter: ${JSON.stringify(request.body)}`,
-        user: response.locals.user.id,investigation: epidemiologyNumber
+        user: response.locals.user.id,investigation: response.locals.epidemiologynumber
     });
     graphqlRequest(UPDATE_LIST_OF_CONTACTS, response.locals, { unSavedContacts: JSON.stringify(request.body)})
         .then((result: any) => {
-            logger.info({
-                service: Service.SERVER,
-                severity: Severity.LOW,
-                workflow,
-                step: 'got response from DB',
-                user: response.locals.user.id,investigation: epidemiologyNumber
-            });
-            isThereDoneContact && sendSavedInvestigationToIntegration(epidemiologyNumber, workflow, response.locals.user.id);
-            response.send(result);
+            if(!result.errors) {
```

Comment:
Is this the check you want to commit? I think that checking if the result structure is as expected is more accurate